droidlet/dashboard/web/src/components/Memory2D/OverlayedObjsPopup.js

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect, useState } from "react";
+import { withStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import Box from "@material-ui/core/Box";
+
+import IconButton from "@material-ui/core/IconButton";
+import ClearIcon from "@material-ui/icons/Clear";
+import Tooltip from "@material-ui/core/Tooltip";
+
+const MAX_TABLE_CELL_WIDTH = 100;
+const MAX_TABLE_CONTAINER_HEIGHT = 220;
+
+const StyledTableCell = withStyles((theme) => ({
```

Comment:
Note that this unconventional method for styling is used because I couldn't get the typical 'consolidate to CSS file + makeStyles' method to work (the styles kept getting overridden). Someone with more experience than me could try to move these methods to make the file more readable